subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -141,8 +141,8 @@ If you'd like to optimize your build so that only out-of-date input files are pr
 [NOTE]
 ====
 There is the link:{javadocPath}/org/gradle/api/tasks/incremental/IncrementalTaskInputs.html[IncrementalTaskInputs] API, which is available in Gradle versions before 5.4.
-The usage of this API is discouraged, since it does not allow querying individual input properties for changes.
-The `IncrementalTaskInputs` API will be deprecated and eventually removed.
+The usage of this API is deprecated, since it does not allow querying individual input properties for changes.
```

Comment:
It is not deprecated because it doesn't allow querying individual properties. It's deprecated because we have a new API (which we also think is better), and we don't want maintain both, no?

Reply:
Better now?